sdk/keyvault/keyvault-keys/package.json

```diff
@@ -72,18 +73,22 @@
     "@azure/core-http": "^1.0.0-preview.1",
     "@azure/core-paging": "^1.0.0-preview.1",
     "@azure/identity": "^1.0.0-preview.1",
+    "@types/query-string": "6.2.0",
     "tslib": "^1.9.3"
   },
   "devDependencies": {
+    "@azure/ms-rest-js": "^1.2.6",
```

Comment:
Didn't we move from "@azure/ms-rest-js" to "@azure/core-http"?
"@azure/core-http" is present in the dependencies.

Reply:
This is true! I thought I couldn't remove it, but turns out I can! Update soon.